LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 - present React Motion authors
```

Comment:
If you want to be specific about copyright holders, I believe Google and Cheng Lou are the copyright holders for the upstream file.  (@bsansouci wrote the original.)

I've added an AUTHORS file to separate the enumeration of copyright holders from the declaration of the license; it's a commonly-followed convention.

Of course, you are free to organize repos you host however you like.  😃